### ISSUE TITLE
Contact Applicant Log- Navigation

### ISSUE THREAD

**Issue by KaraBeach** [bcgov/foi-flow#2684]
* As an IAO analyst
* I want to navigate to the applicant correspondence log easily
* so that I can send, receive and view correspondence with applicants

**Assumptions & Scope**	
What are the assumptions for this story?

Only IAO analysts can view the Applicant Correspondence Log; ministry coordinators cannot see it.

Design wise applicant correspondence will function similar to how Comments does.

IAO analysts need a way to contact applicants.

Traffic will be 1-way. Only emails sent by IAO will appear; applicant replies will be in the Inbox.

Separation of concerns from Comments.

What is IN scope?
* A new Applicant Correspondence Log.

What is NOT in scope?
* Reply comment/notification is not in scope
* Comments
* Searching applicant correspondence and tags will be covered in a separate story.

**Acceptance Criteria**	

**Scenario 1: Left Hand Nav Panel View**
•	GIVEN you are on the review request view of a request
•	WHEN the left side navigation panel appears
•	THEN you will see an option for "Applicant Correspondence"
•	AND if there are any correspondence uploaded to a request
•	THEN the number of correspondence will appear in parentheses (ie: Correspondence (4) )

**Scenario 2: Navigate to Correspondence Page**
•	GIVEN you are on the review request view of a request
•	WHEN you click on the "Applicant Correspondence" panel of the left side navigation panel
•	THEN you will be directed to the correspondence page https://jacklyn808742.invisionapp.com/console/share/3W2O64AP5C/941001461 

**Scenario 3: Correspondence Page View**
•	GIVEN you have clicked on the Applicant Correspondence panel on the left side navigation panel
•	WHEN the Correspondence log populates
•	THEN any Correspondence uploaded to the file will appear in an ordered list, sorted by newest uploads first
•	AND each entry will include: the user that uploaded the file, the date/time the file was uploaded, and the title of the correspondence https://jacklyn808742.invisionapp.com/console/share/3W2O64AP5C/941001461 

...

**Dependencies? What is the impact of this dependency? (If so, link dependency in the ticket, make it visible in a team´s backlog)**

**Validation Rules? (If yes, list here)**	

**Design**
@xxx - please link the Design here

**Definition of Ready**

1. [ ] Is there a well articulated User Story?
2. [ ] Is there Acceptance Criteria that covers all scenarios (happy/sad paths)?
3. [ ] If there is a user interface, is there a design? 
4. [ ] Does the user story need user research/validation?
5. [ ] Does this User Story needs stakeholder approval?
6. [ ] Design / Solution accepted by Product Owner
7. [ ] Is this user story small enough to be completed in a Sprint? Should it be split?
8. [ ] Are the dependencies known/ understood? (technical, business, regulatory/policy)
9. [ ] Has the story been estimated?

**Definition of Done**

1.    [ ] Passes developer unit tests
2.    [ ] Passes peer code review
3.    [ ] If there's a user interface, passes UX assurance
4.    [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
5.	[ ] Confirm Test cases built and succeeding 
6.	[ ] No regression test failures
7.	[ ] Test coverage acceptable by Product Owner
8.	[ ] Ticket ready to be merged to master or story branch
9.	[ ] Developer to list Config changes/ Update documents and designs
10.	[ ] Can be demoed in Sprint Review 
11.	[ ] Tagged as part of a Release
12.   [ ] Feature flagged if required
13.  [ ] Change Management activities done?

**Comment by m-prodan** [comment]
So this one passes QA - however, I'm flagging some issues that we'll likely see in other stories when we try to QA them:

1) Microsoft Edge browsers don't have full functionality - when I try to add correspondence, I am unable to pick a template from the dropdown - this is not a problem in Chrome. This should impact the select template user story

2) There appears to be some weird caching issue happening. I clicked in another file after entering some correspondence, and I could see the correspondence from the previous file, in the new file. When I hit refresh, it cleared and was OK again.

cc: @lmullane @nkan-aot @KaraBeach

**Comment by nkan-aot** [comment]
@m-prodan i will take a look into these issues

**Comment by nkan-aot** [comment]
@m-prodan for number 1, I am not able to reproduce in Edge. Did you have an approved CFR form version in the request you were trying this in? We disabled templates before CFR approval to prevent errors in forms flow since sending Fee Estimate or Outstanding Payment emails generates a payment link, and this will not work if there hasn't been a fee estimate yet. This wasn't in the AC but it was a situation we encountered during development, and anyways it doesn't make sense to send a fee estimate email before there is a fee estiamte
